web/src/components/WithAnalytics/WithAnalytics.tsx

```diff
@@ -5,7 +5,7 @@ const withAnalytics = <P extends object>(Component: React.ComponentType<P>, name
   const FunctionComponent = (props: P) => {
     useEffect(() => {
       AnalyticsService.page(name);
-    }, []);
+    }, [props]);
```

Comment:
For some reason the analytics was getting staggered so I had to add this to capture all of the events